lib/sparse/src/index/search_context.rs

```diff
@@ -134,19 +134,36 @@ impl<'a> SearchContext<'a> {
         filter_condition: &F,
     ) {
         for posting in self.postings_iterators.iter_mut() {
-            for element in posting.posting_list_iterator.remaining_elements() {
+            // index at which the posting list stops contributing to the batch (relative to the batch start)
+            let mut posting_stopped_at = None;
+            for (index, element) in posting
```

Comment:
Nitpick, but wouldn't it be better to call it `offset`. It's the _index_ in the remaining set, not in the full set. Right?

Reply:
You are right, `offset` is a better fit there :+1: 
https://github.com/qdrant/qdrant/pull/3527/commits/3d24a17ef4c2479f7821f0ab2a8fd92eeff67436